### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by 187jjay187**
### Blog app - Add forms

in this milestone I completed the below requirements:

### Project requirements
-  Create a method called `current_user` in your ApplicationController to make current user data available to all controllers.
    -  It will return the first user from the database.
    -  Don't worry about proper authentication at this time. You will add it in the upcoming projects.
- Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user` (use the method that you created in your ApplicationController).
    - Create a comment on behalf of the `current_user` (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/views/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<%= form_with model: @post, url: user_posts_path, method: :post, local: true, scope: :new_post do |form| %>
```
Comment (codecaiine) [inline app/views/new.html.erb]: - [ ] I would like to recommend you create the form for creating posts inside the post view, then you will get **views/posts/new.html.erb** , For each controller, it is very important to associate the directory in the app/views directory which holds the template files that make up the views associated with that controller to display the view that results from each controller action. Please fix it in order to implement all project requirement 👍

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @187jjay187 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good adding form setup ✔️ 
- No rubocop errors ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/new.html.erb (hunk 1 above)
Comment (187jjay187) [reply app/views/new.html.erb]: @codecaiine thank you for the code review I really do appreciate it, and I hope I've met the required changes above to complete this milestone

**Review by TadesseDev — APPROVED** [review]
### Hi @187jjay187 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)

## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**